Inspector/load_commands.c

```diff
@@ -0,0 +1,63 @@
+#include "load_commands.h"
+
+#include <stdio.h>
+#include <mach-o/dyld.h>
+
+char *lc_lookup_by_value(uint32_t value) {
```

Comment:
It's a (well-defined) misfeature of C that you can assign a "const char*" to a "char*". Should probably return "const char*" here.